Server/JokesOnYou.Web.Api/Controllers/JokesController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using JokesOnYou.Web.Api.DTOs;
+using JokesOnYou.Web.Api.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace JokesOnYou.Web.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    [Authorize]
+    public class JokesController : ControllerBase
+    {
+        private readonly IJokesService _jokesService;
+
+        public JokesController(IJokesService jokesService)
+        {
+            _jokesService = jokesService;
+        }
+
+        [AllowAnonymous]
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<JokeReplyDto>>> GetAllJokesAsync()
```

Comment:
Suggestion: No need to name the method async considering no one will be awaiting it

Reply:
There is no need indeed.
I'm just appending Async to all async methods no matter where to make it a habit for myself.
I'd rather have to much information then forgetting it somewhere. 

So I think I'll keep it but, you can change my mind of course.